app/components/modals/index.js

```diff
@@ -0,0 +1,15 @@
+import {mapStateToProps} from '/store'
+import Example from './example'
+
+const MODALS = {
+  Example,
+}
+
+export default mapStateToProps(
```

Comment:
@adriaanwm Want I should move this to `wasmuth`?

Reply:
yeah definitely. But do you think we should use it for a bit first in case any issues come up?